lib/data/student_repository.dart

```diff
@@ -3,7 +3,7 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:open_screener/model/student.dart';
 
 // School table
-String schoolReference = "school";
+String schoolReference = "schools";
```

Comment:
I think its more common to have singular table names.  The reason being something similar to: You put cookies in a cookie jar and not cookies in a cookies jar.